extension/src/main/webapp/WEB-INF/conf/chat/templates/notification/push/ChatPushPlugin.gtmpl

```diff
@@ -1 +1,17 @@
-<%=_ctx.appRes("exoplatform.chat.MentionPlugin", USER, ROOM_NAME)%>
\ No newline at end of file
+<%
+def notificationDescription = "";
+switch(NOTIF_TYPE) {
+ case "ChatMentionNotificationPlugin":
+   notificationDescription =  _ctx.appRes("exoplatform.chat.MentionPlugin", USER, ROOM_NAME);
+   break;
+ default:
+    if(Boolean.parseBoolean(IS_GROUP_CHAT))
+      notificationDescription = "<b>"+ROOM_NAME+"</b><br>"+USER+": "+MESSAGE
+    else
+      notificationDescription = "<b>"+USER+"</b><br>"+MESSAGE
```

Comment:
Shouldn't we add a message like : "USER sent you a new message" ?

Reply:
The goal is to have the same notification we already have on Chat app in which we display a part of the message and the user name when we have a one to one chat room, and the Chat room label, the user who sent the message and a part of the message. 
Thus, we can use it like you suggest as an MVP and we improve it. wdyt @ahamdi

Reply:
OK, let's keep it as is